spec/components/candidate_interface/gcse_qualification_review_component_spec.rb

```diff
@@ -2,25 +2,53 @@
 
 RSpec.describe CandidateInterface::GcseQualificationReviewComponent do
   context 'with the international_gcses flag on' do
-    it 'displays award year, qualification type, grade and institution country' do
+    before do
+      FeatureFlag.activate('international_gcses')
+    end
+
+    it 'displays award year, qualification type, grade, institution country, naric reference and comparable UK qualification' do
```

Comment:
Suggest renaming to `it 'renders a non-uk GCSE equivalent'` (or words to that effect) - the expectations at the end of the `it` block are easily scannable for anyone interested in the detail.

Reply:
Good point. I have changed it. Thanks for the feedback.